src/python/pants/backend/helm/subsystems/helm.py

```diff
@@ -132,6 +133,7 @@ class HelmSubsystem(TemplatedExternalTool):
         advanced=True,
     )
 
+    skip = SkipOption("lint")
```

Comment:
See below about Javac. I suspect this too is a weird option name and should be `--helm-lint-skip` or something